helm-charts/konk-service/templates/delete-apiservice-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Would this be better as a Job?

Reply:
It shouldn't matter, right? Whether it's a job/pod, it will get deleted along with the rest of the helm-chart when the konk-service is deleted.

Reply:
It would change the retry behavior, but this implementation doesn't depend on that.